Content.Shared/Contraband/ContrabandComponent.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Content.Shared.Roles;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Contraband;
+
+/// <summary>
+/// This is used for marking entities that are considered 'contraband' IC and showing it clearly in examine.
+/// </summary>
+[RegisterComponent]
+public sealed partial class ContrabandComponent : Component
+{
+    /// <summary>
+    ///     The degree of contraband severity this item is considered to have.
+    /// </summary>
+    [DataField]
+    public ContrabandSeverity Severity = ContrabandSeverity.Restricted;
+
+    /// <summary>
+    ///     Which departments is this item restricted to?
+    ///     By default, command and sec are assumed to be fine with contraband.
+    ///     If null, no departments are allowed to use this.
+    /// </summary>
+    [DataField]
+    public HashSet<ProtoId<DepartmentPrototype>>? AllowedDepartments = ["Security"];
+}
+
+[Serializable, NetSerializable]
+public enum ContrabandSeverity
+{
```

Comment:
This needs to be a prototype per [Enums vs Prototypes code convention.](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html?highlight=enum#enums-vs-prototypes)